Code/Framework/AzFramework/AzFramework/API/ApplicationAPI.h

```diff
@@ -18,6 +18,7 @@
 #include <AzCore/std/functional.h>
 #include <AzCore/std/parallel/thread.h>
 #include <AzCore/std/string/string.h>
+#include <AzCore/PlatformIncl.h>
```

Comment:
PlatformIncl.h is special in that it includes Windows.h and undefs some of the default WinAPI macros.
Therefore it should be the first include the file.